src/adlmidi_opl3.cpp

```diff
@@ -194,6 +194,13 @@ void OPL3::writeRegI(size_t chip, uint32_t address, uint32_t value)
 #endif
 }
 
+void OPL3::writePan(size_t chip, uint32_t address, uint32_t value)
+{
+#ifndef ADLMIDI_HW_OPL
+    m_chips[chip]->writePan(static_cast<uint16_t>(address), static_cast<uint8_t>(value));
```

Comment:
Don't forget to put ADL_UNUSED () for case of hw-opl, or warnings are will meet you

Reply:
```cpp
#else
    ADL_UNUSED(chip); 
    ADL_UNUSED(address);
    ADL_UNUSED(value);
```